examples/lazy-injection/counter-todo/src/App.tsx

```diff
@@ -0,0 +1,89 @@
+import { Suspense, useReducer, useTransition } from "react"
+import "./App.css"
+import logo from "./logo.svg"
+import { Todos } from "./features/todos/Todos"
```

Comment:
```suggestion
import type { Todos } from "./features/todos/Todos"
```

Reply:
so basically, i should enable consistent-type-imports 😛